### ISSUE TITLE
Enhancements to dc_rights to support more rights statements

### ISSUE THREAD

**Issue by mejackreed** [geoblacklight/geoblacklight-schema#76]

**Comment by mejackreed** [comment]
Maybe rights is similar to https://project-open-data.cio.gov/v1.1/schema/#rights or https://project-open-data.cio.gov/v1.1/schema/#accessLevel

Possible we should also provide a way to add a license for the data?

**Comment by mejackreed** [comment]
http://rightsstatements.org/en/

**Comment by andrewbattista** [comment]
Agreed. In addition to the discussion of this already, I would support adding a place for CC license buttons in the UI when appropriate
